nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/output.md

```diff
@@ -1,5 +1,11 @@
 # {{ cookiecutter.name }}: Output
 
+## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/{{ cookiecutter.short_name }}](https://nf-co.re/{{ cookiecutter.short_name }})
```

Comment:
```suggestion
## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/{{ cookiecutter.short_name }}/usage](https://nf-co.re/{{ cookiecutter.short_name }}/usage)
```

Reply:
Why not actually linking to the page itself?

Reply:
No particular reason..